lib/resources/users.rb

```diff
@@ -213,42 +213,38 @@ def warndays
 
     # implement 'mindays' method to be compatible with serverspec
     def minimum_days_between_password_change
-      deprecated('minimum_days_between_password_change', "Please use: its('mindays')")
+      Inspec.deprecate(:resource_user, 'The user resource `minimum_days_between_password_change` property is deprecated. Please use `mindays`.')
```

Comment:
I'd change all of these to something like :user_resource_serverspec_compat .  Makes the intent and scope more explicit.